daemon/config.cfg

```diff
@@ -12,3 +12,7 @@ level=DEBUG
 [RPCSERVICE]
 host=0.0.0.0
 port=50000
+
+[Spotify_Parameters]
+Client_ID=e9676baa69da445bbd9b691658628b69
+Client_Secret=20727079480e4093a14529aefd950ffa
```

Comment:
I think this id and secret is not proper in the config  file unless they're for development only. So please remove this sensitive data from the config file. Or just don't commit this settings in the branch.